pom.xml

```diff
@@ -93,6 +94,14 @@
 				<version>${webservicesRestVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+			
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>legacyui-omod</artifactId>
```

Comment:
Why do we need this? I think that App Framework should be dropping support for the Legacy UI.